### ISSUE TITLE
With enhanceReducer there are no reducer to call activateReducer on

### ISSUE THREAD

**Issue by Xyzor** [redux-offline/redux-offline#297]
### Are you using `offline()` or `createOffline()` API?
createOffline()

### Store config
```js
import { AsyncStorage } from "react-native";
import { createStore, applyMiddleware, compose, Store } from "redux";
import { persistStore, persistReducer } from "redux-persist";
import { createOffline } from "@redux-offline/redux-offline";
import offlineConfig from "@redux-offline/redux-offline/lib/defaults";
import thunk, { ThunkMiddleware } from "redux-thunk";
/* DEV */
import { composeWithDevTools } from "redux-devtools-extension";

import reducers from "store/reducers";

const persistConfig = {
    key: "root",
    debounce: 500,
    storage: AsyncStorage
};
const { middleware, enhanceReducer, enhanceStore } = createOffline({ ...offlineConfig, persist: false });
const persistedReducer = persistReducer(persistConfig, enhanceReducer(reducers));

export default function configureStore() {
        const store = createStore(
            persistedReducer,
            {},
            composeWithDevTools(
                enhanceStore,
                applyMiddleware(
                    middleware,
                    thunk,
                    require("redux-immutable-state-invariant").default()
                )
            )
        );
    const persistor = persistStore(store, null, () => {
        console.log("Store rehydrated!");
    });
    return { store, persistor };
}
```

### Offline config

The default with `persist: false`

### Are you reporting a bug?

Hopefully not.

### Description

I'm trying to migrate to redux-persist v5 using this [post](https://github.com/redux-offline/redux-offline/issues/119#issuecomment-430923733).
When I start the app i get this error message:

![screenshot_20190213-161551](https://user-images.githubusercontent.com/9139545/52723218-67460300-2fad-11e9-87b0-9a054a94d522.png)

**Comment by calumpeak** [comment]
Hi @Xyzor we don't actively support persist v5 yet so YMMV with older posts, especially if there have been upgrades.
More than happy to help if you can provide a codesandbox link or something like that so I can have a play around with it, as flying blind is difficult here.

**Comment by Xyzor** [comment]
I'll try to create a MCVE this week on snack.

**Comment by Xyzor** [comment]
I probably messed up something.

I created a snack with all the module that I'm currently using and it works just fine with redux-persist v5. Although on snack i could not figure out how to use v5 instead of redux-offline's v4 peer dependency.

[snack](https://snack.expo.io/@fitsolutionskft/redux-offline-sandbox)

Note: I created this snack for a bug report which v5 supposed to fix, but it didn't :cry:  .
